azure-pipelines.yml

```diff
@@ -76,14 +76,15 @@ jobs:
           yarn bundle --no-cache $(sinceArg)
         displayName: bundle
 
+      - script: |
+          yarn workspace @fluentui/docs build
+        displayName: build FUI N* docs
```

Comment:
Are there reasons to have this? 🤔 As I see `yarn bundle` runs build...

![image](https://user-images.githubusercontent.com/14183168/111352650-b414d000-8684-11eb-9a54-fef0aa9849f4.png)

Reply:
Oops, this was left over from another change, I don't think it's needed